realm/realm-library/src/androidTest/java/io/realm/SortTest.java

```diff
@@ -153,19 +153,19 @@ private void checkSortTwoFieldsStringAscendingIntAscending(RealmResults<AllTypes
 
         assertEquals("Adam", results.get(0).getColumnString());
         assertEquals(4, results.get(0).getColumnLong());
-        assertEquals(2, ((TableView) results.getTable()).getSourceRowIndex(0));
+        assertEquals(2, ((TableView) results.getTableOrView()).getSourceRowIndex(0));
```

Comment:
At least I spend some time going in the wrong direction based on it being named `getTable()`, at least with `getTableOrView()` you know it could be both.

But the sooner we get rid of it all together the better.